docs/lib/Components/ModalsPage.js

```diff
@@ -113,6 +121,33 @@ const ModalsPage = () => {
         </PrismCode>
       </pre>
 
+      <h4>Fullscreen Modals</h4>
+
+      <div className="docs-example">
+        <div className="btn-group">
+          <div className="btn">
```

Comment:
I don't think `btn` alone is valid, and/or should we use Button and ButtonGroup?  Is this style copied from other docs?

Reply:
yeah - I copied from another section on the page 😂